### PR TITLE
Feat: Detect Associated PR

### LINKED ISSUE #243: Pull Request Dynamic Status Update
The bot should be able to automatically detect associated PRs with a current bounty in order to detect bounty hunter activity instead of relying on the bounty hunter to manually comment on the issue with an update.

---
              > Do you have any updates @Steveantor? If you would like to release the bounty back to the DevPool, please comment `/unassign`

There's a linked PR

_Originally posted by @Steveantor in https://github.com/ubiquity/bounty-bot/issues/192#issuecomment-1501464389_

### PR DESCRIPTION
**Pull request by Steveantor**
Resolves #243

### CODE REVIEW

File: src/helpers/parser.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  owner: string;
+  repo: string;
+  issue_number: number;
+}
+
+export const gitIssueParser = async ({ owner, repo, issue_number }: GitParser): Promise<boolean> => {
+  try {
+    const { data } = await axios.get(`https://github.com/${owner}/${repo}/issues/${issue_number}`);
+    const dom = parse(data);
+    const devForm = dom.querySelector("[data-target='create-branch.developmentForm']") as HTMLElement;
```
[the remark this answers is not in the document]
Comment (Steveantor) [reply src/helpers/parser.ts:14]: I can confirm that there's no standard REST API for that

[the remark this answers is not in the document]
Comment (rndquu) [reply src/helpers/parser.ts:14]: Yes, unfortunately there is no standard API for getting linked PRs for an issue

File: src/utils/github.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { AxiosResponseHeaders } from "axios";
+// import { AxiosResponseHeaders } from "axios";
```
Comment (0xcodercrane) [inline src/utils/github.ts:1]: lets remove!

**Review by 0xcodercrane — CHANGES_REQUESTED** [review]

**Review by pavlovcik — APPROVED** [review]
Not sure how to test this but the code looks fine. @0xcodercrane please merge this in as soon as you re-review.

**Comment by 0xcodercrane** [comment]
> Not sure how to test this but the code looks fine. @0xcodercrane please merge this in as soon as you re-review.

I will quickly take a deep dive on how to get the linked PR for the issue and merge it

**Pull request merged**